Content.IntegrationTests/Tests/EntityTest.cs

```diff
@@ -70,7 +74,7 @@ public async Task SpawnTest()
                     catch (Exception e)
                     {
                         Logger.LogS(LogLevel.Error, "EntityTest", "Entity '" + prototype.ID + "' threw: " + e.Message);
-                        //Assert.Fail();
```

Comment:
ah yes, the ultimate way to pass tests: commenting things out

Reply:
![image](https://user-images.githubusercontent.com/10968691/90789839-a8884300-e307-11ea-85eb-e00b8cb8e383.png)